extension/app/scripts.babel/app.js

```diff
@@ -19,11 +19,20 @@ document.addEventListener('DOMContentLoaded', _ => {
   const background = chrome.extension.getBackgroundPage();
   const siteNameEl = window.document.querySelector('header h2');
   const resultsEl = document.body.querySelector('.results');
+  const reloadAndRunAllEl = document.body.querySelector('.reload-all');
 
   background.runAudits().then(ret => {
     resultsEl.innerHTML = ret;
   });
 
+  reloadAndRunAllEl.addEventListener('click', () => {
+    background.runAudits({
+      reloadPageAndRunAllTests: true
```

Comment:
maybe call this just `reloadPage`? e.g. we may someday have mobile-only tests (we may already have those), so any gather that requires mobile isn't going to return artifacts if run without the mobile flag, so it won't necessarily be all the tests, just the addition of the ones that require a reload.